docs/generated/checks.md

```diff
@@ -10,7 +10,7 @@ KubeLinter includes the following built-in checks:
 
 **Remediation**: Where possible, remove create access to pod objects in the cluster.
 
-**Template**: [access-to-resources](templates.md#access-to-resources)
+**Template**: [access-to-resources](generated/templates.md#access-to-resources)
```

Comment:
This link is broken. If you go to https://github.com/stackrox/kube-linter/blob/553365517c108e7cb200af7d98527b8716e236ac/docs/generated/checks.md and click that link it will give you https://github.com/stackrox/kube-linter/blob/553365517c108e7cb200af7d98527b8716e236ac/docs/generated/generated/templates.md#access-to-resources that does not exists

Reply:
I'm not sure how GitHub is handling the links. I've built the site locally with docsify and used `docsify serve ./docs` command to check. 

For example, go to https://stackblitz.com/~/github.com/gaurav-nelson/kube-linter/tree/fix-template-links and run `npm i docsify-cli` and then `docsify server ./docs` commands to check.

Reply:
Ah, that's how it's done. How about keeping it working on github and docksify by adding `relativePath: true,` in 
https://github.com/stackrox/kube-linter/blob/553365517c108e7cb200af7d98527b8716e236ac/docs/index.html#L16-L19
Then we only need to change `index.html`

Reply:
I tried this locally. It fixes the check and template links. But if you click another topic, for example, Using KubeLinter it tries to open `http://localhost:3000/#/generated/using-kubelinter` 404.

Reply:
Good catch! So we need to change index and https://github.com/stackrox/kube-linter/blob/553365517c108e7cb200af7d98527b8716e236ac/docs/_sidebar.md to use absolute paths. Then it will not work on github and IDE but probably it's fine as nobody will use this file

Reply:
Hmm, it's working for me with this patch
```diff
diff --git a/docs/_sidebar.md b/docs/_sidebar.md
index 3457af1..3412f76 100644
--- a/docs/_sidebar.md
+++ b/docs/_sidebar.md
@@ -1,5 +1,5 @@
 * [Introduction](/)
-* [Using KubeLinter](using-kubelinter.md)
-* [Configuring KubeLinter](configuring-kubelinter.md)
-* [KubeLinter checks](generated/checks.md)
-* [KubeLinter templates](generated/templates.md)
\ No newline at end of file
+* [Using KubeLinter](/using-kubelinter.md)
+* [Configuring KubeLinter](/configuring-kubelinter.md)
+* [KubeLinter checks](/generated/checks.md)
+* [KubeLinter templates](/generated/templates.md)
diff --git a/docs/index.html b/docs/index.html
index dc7b2eb..deab721 100644
--- a/docs/index.html
+++ b/docs/index.html
@@ -27,6 +27,7 @@
         errorText  : 'Error',
         successText: 'Copied'
       },
+      relativePath: true,
       'flexible-alerts': {
         style: 'flat'
       },

```